omero/downloads/ldap/setdn.out

```diff
@@ -1,4 +1,4 @@
-usage: bin/omero ldap setdn [-h] [--user-id USER_ID] [--user-name USER_NAME]
```

Comment:
This is another autogen file. argparse keeps the arguments aligned which led to the conflict. Reverting this to the original.